packages/ag-charts-community/src/chart/series/cartesian/scatterSeries.ts

```diff
@@ -170,7 +170,8 @@ export class ScatterSeries extends CartesianSeries<Group, ScatterNodeDatum> {
         const nodeData: ScatterNodeDatum[] = [];
 
         const font = label.getFont();
-        for (const { values, datum } of processedData.data ?? []) {
+        for (let i = 0; i < processedData.data.length; i++) {
+            const { values, datum } = processedData.data[i];
```

Comment:
Nit: Can we just revert this line to remove the use of `i`?

Reply:
Well spotted. Yes of course. I'll revert this.